sql/explain.go

```diff
@@ -48,9 +51,20 @@ func (p *planner) Explain(n *parser.Explain) (planNode, error) {
 	if err != nil {
 		return nil, err
 	}
-	plan, err = markExplain(plan, mode)
-	if err != nil {
-		return nil, fmt.Errorf("%v: %s", err, n)
+	switch mode {
+	case explainDebug:
+		plan, err = markExplain(plan, mode)
```

Comment:
the naming is getting a bit fuzzy with `explainNode`. perhaps a good time to rename